pkg/install/install.go

```diff
@@ -136,10 +136,15 @@ func Install(ctx context.Context, image string, opts *Options) error {
 	}
 
 	// Validate E-Mail address provided for Let's Encrypt registration
-	if finalConfForValidation.LetsEncryptEmail != "" || *finalConfForValidation.LetsEncrypt == "enabled" {
-		email := finalConfForValidation.LetsEncryptEmail
+	email := ""
+	if finalConfForValidation.LetsEncryptEmail != "" {
+		email = finalConfForValidation.LetsEncryptEmail
+	} else {
+		email = opts.Config.LetsEncryptEmail
+	}
+	if email != "" || *finalConfForValidation.LetsEncrypt == "enabled" {
```

Comment:
Is it valid to have an empty email address here?

If no, then we can remove the `email != ""` check here.

Reply:
I believe this is in there to handle when lets encrypt is disabled but a email is provided we still want to validate it.

Reply:
Got it. Makes sense. Thanks @jsilverio22!